po/summary/introduction.zh-cn.po

```diff
@@ -0,0 +1,53 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: The Book of sbt\n"
+"POT-Creation-Date: 2024-10-30T03:50:36-04:00\n"
+"PO-Revision-Date: 2024-10-30 04:53-0400\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: none\n"
+"Language: zh_CN\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"X-Generator: Poedit 3.5\n"
+
+#: src/reference/index.md:1
+msgid "The Book of sbt (Draft)"
+msgstr "sbt 记 (草稿)"
+
+#: src/reference/index.md:4
+msgid ""
+"This is a draft documentation of sbt 2.x that is yet to be released. While "
+"the general concept translates to sbt 1.x, details of both 2.x and this doc "
+"are subject to change."
+msgstr ""
+"这是关于 sbt 2.x 的草稿文档，目前尚未发布。虽然总体概念可以应用到 sbt 1.x，"
+"但 2.x 及本文档的详细内容可能会有所变动。"
```

Comment:
This is a demo `.po` file I created by machine translating using ChatGPT. Post merge, this should be checked by human.